modular_skyrat/master_files/code/modules/uplink/uplink_items.dm

```diff
@@ -369,7 +369,7 @@
 	desc = "Fight the power with this frontline combatant kit, featuring armor and armaments commonly utilized by assault operative teams."
 	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/metaops
 	cost = 23
-	progression_minimum = 35 MINUTES //+10 minutes because of the bulldog
+	progression_minimum = 60 MINUTES //+10 minutes because of the bulldog
```

Comment:
please update comments for the added time from the baseline time (25 minutes)

Reply:
Resolved, I think?